src/components/Chat/Chat.js

```diff
@@ -44,8 +44,7 @@ const ChatComposerWrapper = styled.div`
   padding: 0;
   display: flex;
   flex-direction: column;
-  max-height: ${props => props.theme.screen.medium.height}px;
-  min-height: ${props => props.theme.screen.small.height}px;
+  height: 340px;
   @media (max-width:640px) {
```

Comment:
1. Can we squash and merge 2 commits into 1 commit. 
2. The video seems like the `height` fix is not working as we expected. I think we wanted the height of the widget to be fixed and not increase as we add more messages ?